dao/dao.cabal

```diff
@@ -1,96 +1,94 @@
 cabal-version: 2.4
 name:          dao
 version:       1.0.0.0
-license: MIT
+license:       MIT
 author:        Jonathan Fischoff
 maintainer:    jonathan@Canonicalllc.com
 build-type:    Simple
-copyright: 2021 Canonical LLC
+copyright:     2021 Canonical LLC
```

Comment:
Can we change this to "2024 Yaadlabs Ltd."?

Reply:
Sure, I'll do that in a moment